src/app/globals.css

```diff
@@ -1,4 +1,54 @@
-:root {
+* {
```

Comment:
I assumen it is next.js default css, if it is ok for you you can leave it, but if not just remove it !